src/rard/research/tests/views/test_antiquarian.py

```diff
@@ -181,3 +182,58 @@ def test_ordered_queryset(self):
         qs = view.get_queryset()
         self.assertEqual(2, len(qs))
         self.assertEqual([a.pk for a in qs.all()], [a2.pk, a1.pk])
+
+
+class TestAntiquarianUpdateIntroductionView(TestCase):
+    def setUp(self):
+        self.antiquarian = Antiquarian.objects.create()
+        self.url = reverse("antiquarian:delete", kwargs={"pk": self.antiquarian.pk})
```

Comment:
Why antiquarian:delete? Shouldn't it be update_introduction? Especially given you use self.url in test_update_intro below.
```suggestion
        self.url = reverse("antiquarian:update_introduction", kwargs={"pk": self.antiquarian.pk})
```

Reply:
it should...somewhat worrying this wasn't caught by the test